cmd/storj-admin/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"storj.io/common/fpath"
+	"storj.io/private/cfgstruct"
+	"storj.io/private/process"
+	_ "storj.io/storj/private/version" // This attaches version information during release builds.
```

Comment:
I started thinking that maybe the admin api can return the satellite version with every request that way this tool can detect whether it's the correct version. I think it's easier that way to force people to use the version meant for the particular satellite, rather than silently getting something wrong. At least until we have a stable API.

Reply:
Yeah, thats something i want to implement as well. currently this tool is necessary for creating big chunks of users.
Will do my best to implement the features over time 👍